janusgraph-grpc/src/main/proto/janusgraph/schema/v1/schema_manager.proto

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package janusgraph.schema.v1;
+
+import "janusgraph/types/v1/graph_types.proto";
+import "janusgraph/types/v1/schema_types.proto";
+
+option java_multiple_files = true;
+option java_package = "org.janusgraph.graphdb.grpc.schema";
+
+
+message GetVertexLabelByNameRequest {
+    janusgraph.types.v1.JanusGraphContext context = 1;
+    string name = 2;
+}
+
+message GetVertexLabelByNameResponse {
+    janusgraph.types.v1.VertexLabel vertex_label = 2;
+}
+
+service SchemaManagerService {
+    rpc GetVertexLabelByName(GetVertexLabelByNameRequest) returns (GetVertexLabelByNameResponse);
```

Comment:
In order to get vertex label by name, one has to:
1) Create a JanusGraphManagerClient to fetch JanusGraphContext
2) Create a SchemaManagerClient and use the JanusGraphContext fetched in step 1) to get vertex label

I am wondering if there is such a need. Can we simply have one service and one client?

Reply:
At the moment, i think that should be okay to have just one service. But it would become bigger over time. 

My idea of split it into two client would allow to set the context just once.

Reply:
JanusGraphManagerClient could be later used for CFG components

Reply:
Sounds fair.